scripts/activate.sh

```diff
@@ -1,5 +1,11 @@
 #!/bin/bash
 set -Eeuo pipefail
 export CARGO_TARGET_DIR="target/pixi"
+
+# on linux we need to set these rust flags
```

Comment:
This comment isn't _really_ true as we don't need the flags. We only need it for extra speed with `mold`! :)